build-system/babel-plugins/babel-plugin-amp-mode-transformer/index.js

```diff
@@ -41,11 +41,14 @@ module.exports = function ({types: t}) {
         }
         specifiers.forEach((specifier) => {
           if (specifier.imported && specifier.imported.name === 'getMode') {
-            const filepath = resolve(
-              dirname(state.file.opts.filename),
-              source.value
+            const filepath = relative(
+              join(__dirname, '../../../'),
+              resolve(dirname(state.file.opts.filename), source.value)
             );
-            if (filepath.endsWith('/amphtml/src/mode')) {
+            if (
+              filepath.endsWith('src/mode') ||
+              filepath.endsWith('src\\mode')
+            ) {
```

Comment:
I think this code can be simplified if you use [`path.posix`](https://nodejs.org/api/path.html#path_windows_vs_posix), which returns consistent posix-style paths no matter what OS you're running on.